ase/src/main/java/com/ase/ase/rest/controller/OverviewController.java

```diff
@@ -2,25 +2,20 @@
 
 import com.ase.ase.dao.CasesTimelineRepository;
 import com.ase.ase.rest.response.Overview;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 
 @RestController
-@RequestMapping(
-        method = RequestMethod.GET,
-        path = "/overview",
-        produces = MediaType.APPLICATION_JSON_VALUE
-)
+@RequestMapping("/overview")
```

Comment:
You need the @CrossOrigin annotation for the class and method. Look how it done in the other controllers.